logexp/README.md

```diff
@@ -0,0 +1,40 @@
+# LogExp
+
+LogExp is a tool that runs post-test on our kubernetes test clusters.
+It does the job of computing the set of logfiles to be exported (based on the
+node type (master/node), cloud provider, and the node's system services),and
+then actually exports them to the GCS path provided to it.
+
+The following flags need to be supplied to the tool:
+* --node-name (Name of the node this log exporter is running on)
+* --gcs-artifacts-path (Path to the GCS artifacts dir under which to upload logs, for eg: gs://my-logs-bucket/logs)
+* --kubernetes-provider (Cloud provider for this node (gce/gke/aws/kubemark/..))
```

Comment:
Seems like we are using --cloud-provider (Ref: https://github.com/kubernetes/kubernetes/blob/master/hack/verify-flags/known-flags.txt#L100)
Changed it now. Is it fine?